website/docs/api/Humanizer.NumberToWordsExtension.md

```
@@ -11,6 +11,70 @@ public static class NumberToWordsExtension
 ### Methods
 
+<a name='Humanizer.NumberToWordsExtension.ToIndianWords(thisint,Humanizer.IndianScaleStyle)'></a>
+
+## NumberToWordsExtension\.ToIndianWords\(this int, IndianScaleStyle\) Method
+
+Converts the given value to Indian English cardinal words using the selected large\-number vocabulary\.
+
+```csharp
+public static string ToIndianWords(this int number, Humanizer.IndianScaleStyle scaleStyle=Humanizer.IndianScaleStyle.NamedScales);
+```
+#### Parameters
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)